playwright.config.ts

```diff
@@ -52,7 +52,7 @@ if (IS_EMBED_REACT_TEST) {
 
 const config: PlaywrightTestConfig = {
   forbidOnly: !!process.env.CI,
-  retries: 2,
+  retries: process.env.CI ? 2 : 0,
```

Comment:
I would like tests to on first try locally so that failure reason can be seen quickly.

Reply:
Why not disable retires altogether so tests fail fast on CI?

Reply:
Because on CI it's too flaky to run without retries due to parallel runners. On dev it's more likely you are running only a single test or a few.